zeg/schemata/spec.yaml

```diff
@@ -187,6 +187,46 @@ anyOf:
       - required:
         - imageset_type
         - container_name
+# multiple image sources collection
+- allOf:
+  - $ref: '#/definitions/collection'
+  - collection_version:
```

Comment:
should this just be `version`? to be consistent with cloud

Reply:
This is the name we discussed to be and it's only used in the config file for user so it doesn't really matter.